src/manager.cc

```diff
@@ -46,21 +46,22 @@ Manager::Manager() :
 
   m_hashQueue = new HashQueue(&m_main_thread_disk);
   m_hashQueue->slot_has_work() =
-    std::bind(&thread_base::send_event_signal,
-              &m_main_thread_main,
-              m_main_thread_main.signal_bitfield()->add_signal(std::bind(&HashQueue::work, m_hashQueue)),
-              std::placeholders::_1);
+    [this](auto arg) {
+      auto signal = m_main_thread_main.signal_bitfield()->add_signal(
+        [this]() { return m_hashQueue->work(); });
```

Comment:
Don't split lines this short in this way.

Reply:
Did this in a more readable way I think.